syncer/syncer.go

```diff
@@ -2334,6 +2337,16 @@ func (qec *queryEventContext) String() string {
 		qec.ddlSchema, qec.originSQL, startLocation, currentLocation, lastLocation, shardingReSync, needHandleDDLs)
 }
 
+func genrateTransferColumn(rows [][]interface{}, transferCol, transferVal []string) {
```

Comment:
```suggestion
func genrateTransferColumn(rows [][]interface{})rows[][]interface{} {
    transferCol, transferVal := utils.FetchTableTransferColumnRule(s.tableRouter, sourceTable.Schema, sourceTable.Name, s.cfg.SourceID)
    // other logic
}
```
perfer to move `transferCol, transferVal := utils.FetchTableTransferColumnRule(s.tableRouter, sourceTable.Schema, sourceTable.Name, s.cfg.SourceID)` in to func `genrateTransferColumn` and return new rows after `genrateTransferColumn`, because change `rows [][]interface` in a func may cause confused when outer space use some thing like 

```golang
row = rows[0]
genrateTransferColumn(rows)
//after genrateTransferColumn it's ok to use rows[0] but origin row is different from rows[0]
```